lib/auth/auth_with_roles.go

```diff
@@ -56,8 +56,24 @@ func (a *ServerWithRoles) actionWithContext(ctx *services.Context, namespace str
 	return a.context.Checker.CheckAccessToRule(ctx, namespace, resource, action, false)
 }
 
-func (a *ServerWithRoles) action(namespace string, resource string, action string) error {
-	return a.context.Checker.CheckAccessToRule(&services.Context{User: a.context.User}, namespace, resource, action, false)
+type actionConfig struct {
+	quiet bool
+}
+
+type actionOption func(*actionConfig)
+
+func quiet(value bool) actionOption {
```

Comment:
This flag seems unnecessary (what does it really mean in the bigger picture?) Returning available to the user requests seems like a default and expected behavior (similarly to GetNodes).